smithy-openapi/src/test/resources/software/amazon/smithy/openapi/fromsmithy/security/http-api-key-bearer-security.openapi.json

```diff
@@ -0,0 +1,35 @@
+{
+    "openapi": "3.0.2",
+    "info": {
+        "title": "Service",
+        "version": "2006-03-01"
+    },
+    "paths": {
+        "/": {
+            "get": {
+                "operationId": "Operation",
+                "responses": {
+                    "200": {
+                        "description": "Operation response"
+                    }
+                }
+            }
+        }
+    },
+    "components": {
+        "securitySchemes": {
+            "smithy.api.httpApiKeyAuth": {
+                "type": "http",
+                "description": "ApiKey authentication semantics via 'Authorization' header",
+                "name": "Authorization",
+                "in": "header",
+                "scheme": "ApiKey"
+            }
+        }
+    },
+    "security": [
+        {
+            "smithy.api.httpApiKeyAuth": [ ]
```

Comment:
I'm unsure if the ApiKey auth qualifier is correct being instead of one for bearer auth.

Reply:
I think this makes sense. If nothing else, it prevents conflicts if both `@httpApiKey` and `@httpBearerAuth` are used in a single service.